src/utils/helpers.js

```diff
@@ -2,8 +2,10 @@
  * Get value of an object property/path even if it's nested
  */
 export function getValueByPath(obj, path) {
-    const value = path.split('.').reduce((o, i) => o[i], obj)
-    return value
+    if (typeof obj === 'object') {
```

Comment:
I think it's no correct add this check

Reply:
I wasn't sure if this helper might ever have a type passed in that would throw an error when accessing the index. I think this would happen in case of int being passed in

Reply:
In case of string or other it'll return null value

Reply:
thank you! updating to reflect